### PR TITLE
fix: Update data.json

### PR DESCRIPTION
**Pull request by Gur-jot**
please add my Readme File also

### CODE REVIEW

**Review by schmelto — CHANGES_REQUESTED** [review]
Please add yourself at the end of the data.json file

**Review by CodingSpecies — CHANGES_REQUESTED** [review]
> 
> 
> please add my Readme File also

@Gur-jot your readme will be added, as you gave the image url to it. 
Also, see the changes @schmelto reviewed.

**Comment by Gur-jot** [comment]
@schmelto I just add myself at the end I hope it is right now

File: data.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+
```
Comment (schmelto) [inline data.json:2]:
```suggestion
```

File: data.json [hunk 2]
```diff
@@ -1385,4 +1386,11 @@
     "imageUrl": "https://user-images.githubusercontent.com/73016212/135509959-87f8b8e4-9fab-4247-bae4-d965fca5ddbc.png",
     "issueNumber": 742
   }
+
```
Comment (schmelto) [inline data.json:1389]:
```suggestion
```

File: data.json [hunk 3]
```diff
@@ -1385,4 +1386,11 @@
     "imageUrl": "https://user-images.githubusercontent.com/73016212/135509959-87f8b8e4-9fab-4247-bae4-d965fca5ddbc.png",
     "issueNumber": 742
   }
```
Comment (schmelto) [inline data.json:1388]:
```suggestion
  },
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json [hunk 4]
```diff
@@ -1385,6 +1385,12 @@
     "imageUrl": "https://user-images.githubusercontent.com/73016212/135509959-87f8b8e4-9fab-4247-bae4-d965fca5ddbc.png",
     "issueNumber": 742
   },
+  {
+    "name": "Gurjot Singh",
+    "githubUsername": "gur-jot",
```
Comment (Panquesito7) [inline data.json:1390]:
```suggestion
    "githubUsername": "Gur-jot",
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**